### ISSUE TITLE
Module not found

### ISSUE THREAD

**Issue by BobobUnicorn** [mvoidex/hsdev#25]
I'm rather confused. I have a project with the following cabal/hs files:
https://gist.github.com/BobobUnicorn/c854f0d553b90fc2e815

Running the commands:
>hsdev start
>hsdev scan --project .
>hsdev modules --project . --pretty

gives the following modules:

```
[
    {
        "location": {
            "project": "/home/kchoi/Documents/testproj/testproj.cabal",
            "file": "/home/kchoi/Documents/testproj/app/Main.hs"
        },
        "name": "Main"
    },
    {
        "location": {
            "project": "/home/kchoi/Documents/testproj/testproj.cabal",
            "file": "/home/kchoi/Documents/testproj/src/Lib.hs"
        },
        "name": "Lib"
    },
    {
        "location": {
            "project": "/home/kchoi/Documents/testproj/testproj.cabal",
            "file": "/home/kchoi/Documents/testproj/test/Spec.hs"
        },
        "name": "Main"
    }
]
```

but running:
>hsdev check --file app/Main.hs --pretty

gives:
```
{
    "error": "Could not find module ‘Lib’\nUse -v to see a list of the files searched for.\n",
    "details": {}
}
```

Any ideas?

**Comment by mvoidex** [comment]
Thanks, it's a bug. I'll fix it soon. As a workaround, you can change `hs-source-dirs` for `testproj-exe` to `app, src`.

**Comment by mvoidex** [comment]
Must be fixed in `0.1.7.0`

**Comment by BobobUnicorn** [comment]
Awesome, I'll try it out.
